src/plugins/vis_type_timeseries/server/plugin.ts

```diff
@@ -111,6 +114,11 @@ export class VisTypeTimeseriesPlugin implements Plugin<VisTypeTimeseriesSetup> {
           requestContext.core.elasticsearch.client.asCurrentUser
         );
       },
+      getFieldFormatsService: async (uiSettings) => {
+        const [, { data }] = await core.getStartServices();
+
+        return await data.fieldFormats.fieldFormatServiceFactory(uiSettings);
```

Comment:
nitpick: do we need this `await` here?

Reply:
yes, `fieldFormatServiceFactory` returns a Promise

![image](https://user-images.githubusercontent.com/54894989/132860888-bdaf4fc0-138b-472e-b148-3d87046d03d3.png)

Reply:
when you have an async function defined you don't need to await on what you're returning :) It will return promise anyway. https://eslint.org/docs/rules/no-return-await

Reply:
returning a `Promise` within an async function makes it a `Promise<Promise<typeof result>>` that gets squashed into a `Promise<typeof result>` anyway (it's a `Promise` property to not wrap other Promises).

Reply:
of course! my bad, just fixed that